packages/app-bridge/src/react/useGuidelineActions.ts

```diff
@@ -173,16 +175,31 @@ export const useGuidelineActions = (appBridge: AppBridgeTheme) => {
     );
 
     const createPage = useCallback(
-        async (documentPage: CreateDocumentPage) => {
+        async (documentPage: DocumentPageRequestCreate) => {
```

Comment:
```suggestion
        async (documentPage: CreateDocumentPage) => {
```
Why changing the name? It is less good than the previous one

Reply:
It makes it harder to read so I am up for dedicated types